modules/aws_acm_certificate_arn/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "arn" {
+  value = "${element(concat(data.aws_acm_certificate.cert.*.arn, list("")), 0)}"
```

Comment:
About this change: https://www.terraform.io/upgrade-guides/0-11.html#referencing-attributes-from-resources-with-count-0